Makefile

```
@@ -1,10 +1,46 @@
-build:
-	$(MAKE) -C testdata/hello-go build
+.PHONY: all clean tests lint build format benchmarks
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)